nautobot/utilities/templatetags/helpers.py

```diff
@@ -679,3 +679,16 @@ def custom_branding_or_static(branding_asset, static_asset):
     if settings.BRANDING_FILEPATHS.get(branding_asset):
         return f"{ settings.MEDIA_URL }{ settings.BRANDING_FILEPATHS.get(branding_asset) }"
     return StaticNode.handle_simple(static_asset)
+
+
+@library.filter()
+@register.filter()
+def render_related_name_and_color(model, field_name):
```

Comment:
Given that this doesn't actually need model at all (only obj) what if we changed it to just take the obj directly as input? 

```
{{ object.status | render_name_and_color }}
{{ object.role | render_name_and_color }}
```

Actually, what if we took this even a step further, and just made this an enhancement to the existing `hyperlinked_object` filter (i.e. handle the case where the object has a `color` field)?

Reply:
Changing the '`hyperlinked_object`' to add button styling for `objs` with `color` field has an effect on the breadcrumb. That is, breadcrumbs of roles and status, or anything with a `color` field, appears as a button.

Reply:
Ah, makes sense. Thanks.